### PR TITLE
docs: fix invalid variable and missing semi-colon

### PR DESCRIPTION
**Pull request by odahcam**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngxs/store/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

The example snippet is wrong.

Issue Number: N/A

## What is the new behavior?

Snippet fixed.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: docs/recipes/authentication.md [hunk 1]
```diff
@@ -40,8 +40,8 @@ export class AuthState {
   @Action(Login)
   login({ patchState }: StateContext<AuthStateModel>, { payload }: Login) {
     return this.authService.login(payload).pipe(tap((result: { token: string }) => {
-      patchState({ token, username: payload.username });
-    }))
+      patchState({ result.token, username: payload.username });
```
Comment (eranshmil) [inline docs/recipes/authentication.md]: This is a syntax error.
Should be `token: result.token`.

Comment (splincode) [reply docs/recipes/authentication.md]: Good catch ))

Comment (odahcam) [reply docs/recipes/authentication.md]: When the fix not fix. hehe

File: docs/recipes/authentication.md [hunk 2]
```diff
@@ -39,9 +39,9 @@ export class AuthState {
 
   @Action(Login)
   login({ patchState }: StateContext<AuthStateModel>, { payload }: Login) {
-    return this.authService.login(payload).pipe(tap((result: { token: string }) => {
+    return this.authService.login(payload).pipe(tap(({ token }: { token: string }) => {
```
Comment (arturovt) [inline docs/recipes/authentication.md]: Could you change back to `result: { token: string }`? We prefer using the simple syntax in the docs :)

**Comment by splincode** [comment]
https://github.com/ngxs/store/pull/1013

**Pull request closed without merging**